packages/swingset-runner/test/test-demo.js

```diff
@@ -44,6 +44,6 @@ test('run encouragmentBot demo with lmdb', async t => {
   innerTest(t, '--lmdb');
 });
 
-test('run encouragmentBot demo without SES', async t => {
-  innerTest(t, '--no-ses');
+test('run encouragmentBot demo with default', async t => {
```

Comment:
This is not really right.  The test was literally testing the non-SES case without caring which DB was used.  If that case no longer exists, just take the test out.

Reply:
I didn't see any other test that omits a `--*db` flag.. don't we want to exercise the presence of a default choice? Or is this a "you must pick one" not-really-an-option "option" flag?

Reply:
The test was not a test of what happens if you omit a `--*db` flag, it was a test of what happens if you run outside SES.  There's really not much point to using this particular test just to test command line parsing.  If it made sense to have such a test, that would be a different test (in particular, it wouldn't need to run a couple of vats and have them do arbitrary stuff whose output you check).  But I don't see much value to writing such a test.

Reply:
Ok, I'm going to respectfully disagree and retain it, since a test of the default would catch an error that I've made personally more than once, where I didn't add something like the `let dbMode = '--filedb';` line, and it worked fine from tests where I was providing a specific choice, but then failed in the common case where I didn't provide any. I agree that we could have a separate test that only exercised command-line parsing, but I also agree with what I think you're feeling that such a test would be kind of annoying to write and pretty low-value. Whereas this test is already right here and it's prety easy to just rename, and it seems to only take a second or two to run.